snap/snapcraft.yaml

```diff
@@ -40,20 +41,24 @@ parts:
   www-static:
     plugin: dump
     source: ./static
-    organize:
```

Comment:
Not sure I understand the reason for the change, looking at the reference here https://snapcraft.io/docs/snapcraft-yaml-reference it says that for organise 
>the key represents the path of a file inside the part and the value represents how the file is going to be staged.

Shouldn't the static folder be present when building the snap?

Reply:
Reading your message on CS Private again, this is because we can't organize a folder I guess?

Reply:
Yeah pretty much, that's why it was broken, so when I moved it manually, all good